src/Sylius/Behat/Context/FeatureContext.php

```diff
@@ -0,0 +1,181 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Context;
+
+use Doctrine\ORM\EntityManagerInterface;
+use SensioLabs\Behat\PageObjectExtension\Context\PageObjectContext;
+use Behat\Symfony2Extension\Context\KernelAwareContext;
+use Sylius\Component\Core\Test\Services\SharedStorage;
+use Symfony\Component\HttpKernel\KernelInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Doctrine\ORM\EntityNotFoundException;
+use Behat\MinkExtension\Context\MinkAwareContext;
+use Behat\Mink\Mink;
+use Behat\Mink\Session;
+use Doctrine\Common\DataFixtures\Purger\ORMPurger;
+use Behat\Mink\WebAssert;
+
+/**
+ * @author Arkadiusz Krakowiak <arkadiusz.krakowiak@lakion.com>
+ */
+class FeatureContext extends PageObjectContext implements MinkAwareContext, KernelAwareContext
+{
+    /**
+     * @var ContainerInterface
+     */
+    protected $container;
+
+    /**
+     * @var KernelInterface
+     */
+    protected $kernel;
+
+    /**
+     * @var Mink
+     */
+    protected $mink;
+
+    /**
+     * @var array
+     */
+    protected $minkParameters;
+
+    /**
+     * @var SharedStorage
+     */
+    protected $clipboard;
+
+    /**
+     * @var EntityManagerInterface
+     */
+    protected $entityManager;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function setMink(Mink $mink)
+    {
+        $this->mink = $mink;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function setMinkParameters(array $parameters)
+    {
+        $this->minkParameters = $parameters;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function setKernel(KernelInterface $kernel)
+    {
+        $this->kernel = $kernel;
+    }
+
+    /**
+     * @BeforeScenario
+     */
+    public function purgeDatabase()
+    {
+        if (null === $this->entityManager) {
+            $this->entityManager = $this->getService('doctrine.orm.entity_manager');
+        }
+        $this->entityManager->getConnection()->getConfiguration()->setSQLLogger(null);
+
+        $purger = new ORMPurger($this->entityManager);
+        $purger->purge();
+
+        $this->entityManager->clear();
+    }
+
+    /**
+     * @BeforeScenario
+     */
+    public function setClipboard()
+    {
+        $this->clipboard = $this->getService('sylius.behat.shared_storage');
+    }
+
+    /**
+     * @BeforeScenario
+     */
+    public function setEntityManager()
```

Comment:
Won't it be better to have one step for whole database management? We should consider merging this step with * purgeDatabase*